Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Ammunition/explosives.yml

```
@@ -1,4 +1,21 @@
-﻿# Shells that used for friendship and duster
+﻿- type: entity
+  id: GrenadeFragLight
+  name: frag grenade
+  parent: BaseGrenade
+  components:
+  - type: CartridgeAmmo
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)